lib/vagrant/pre-rubygems.rb

```diff
@@ -1,34 +0,0 @@
-# This file is to be loaded _before_ any RubyGems are loaded. This file
-# initializes the Bundler context so that Vagrant and its associated plugins
-# can load properly, and then execs out into Vagrant again.
-
-require_relative "shared_helpers"
-
-if defined?(Bundler)
-  require "bundler/shared_helpers"
-  if Bundler::SharedHelpers.in_bundle? && !Vagrant.very_quiet?
-    puts "Vagrant appears to be running in a Bundler environment. Your "
-    puts "existing Gemfile will be used. Vagrant will not auto-load any plugins"
-    puts "installed with `vagrant plugin`. Vagrant will autoload any plugins in"
-    puts "the 'plugins' group in your Gemfile. You can force Vagrant to take over"
-    puts "with VAGRANT_FORCE_BUNDLER."
-    puts
```

Comment:
@gildegoma Hi! Thanks very much for the feedback. I did miss that section in the README file and will need to update it. But to the bigger issue around the behavior of package installation vs source installation around plugins, I'm really happy to get this feedback. 

I'm trying to make the behavior of Vagrant from a package installation context and a source installation running via Bundler context function in mostly the same way. This unexpected breakage was "expected" from an implementation point of view, but not intentional from a usability point of view for contributors. My apologies!

I would like for plugins to remain auto loading regardless of the running context (so I'll definitely update that portion of the README, and the relevant section in the website docs). I am curious on thoughts around context specific plugins files and installations. It was something I was considering originally, and perhaps might be worthwhile. This would allow switching from a package installation context to a source installation context without needing to run a `plugin repair` or `plugin expunge --reinstall`.

Any more thoughts/feedback is very welcome. Cheers!